fontra2fontir/src/fontra.rs

```diff
@@ -78,7 +180,7 @@ pub(crate) struct FontraSource {
     #[serde(rename = "layerName")]
     pub(crate) layer_name: String,
     #[serde(default)]
-    pub(crate) location: HashMap<Tag, f64>,
+    pub(crate) location: HashMap<String, f64>,
```

Comment:
if this `String` has any specific meaning (is it supposed to be a tag, or a human readable name, or what?) a comment would be useful.

Reply:
I think it'll help if I newtype some of the name fields that are used as identifiers. `HashMap<AxisName, f64>`, much like the `<Tag, f64>` version. Aside: I wish it was the tag.

Reply:
I will do this in a followon